pkg/apis/core/types.go

```diff
@@ -2138,31 +2138,31 @@ const (
 	PullIfNotPresent PullPolicy = "IfNotPresent"
 )
 
-// ResourceResizePolicy specifies how Kubernetes should handle resource resize.
```

Comment:
CC: @marquiz, was it you we discussed it with when talking about QoS-class resources KEP: https://github.com/kubernetes/enhancements/issues/3008. Cannot recall now. Maybe you have some naming input here

Reply:
No I don't remember discussing this aspect of in-place updates in context of the QoS resources KEP. I don't think we've got this deep in the details there, yet.